packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py

```diff
@@ -191,3 +196,42 @@ async def _call(
         service_key=service_key,
         service_version=service_version,
     )
+
+
+@log_decorator(_logger, level=logging.DEBUG)
+async def batch_get_my_services(
```

Comment:
This also should work, instead of having an internal wrapper.
```suggestion
@log_decorator(_logger, level=logging.DEBUG)
@validate_call(config={"arbitrary_types_allowed": True})
async def batch_get_my_services(
```